src/psyclone/tests/domain/lfric/lfric_psyir_test.py

```diff
@@ -443,3 +445,41 @@ def test_vector_fields(symbol, parent_symbol, dims, attribute_map):
     lfric_symbol = symbol("symbol", dims, *args)
     assert isinstance(lfric_symbol, parent_symbol)
     assert lfric_symbol.name == "symbol"
+
+
+def test_add_lfric_precision_symbol():
+    ''' Test that the add_lfric_precision_symbol() routine rejects invalid
+    precision symbols and otherwise works as expected. '''
+    table = SymbolTable()
+    # Unrecognised name of precision.
+    with pytest.raises(ValueError) as err:
+        lfric_psyir.add_lfric_precision_symbol(table, "r_wrong")
+    # An unsupported precision name. We have to monkeypatch the list of
```

Comment:
That comment seems outdated, no monkeypatch. And even then the comment is add an odd space - between the 'with' and assert, while not really applicable to this.

Reply:
Oops. Removed.